src/scss/lr7-classic-theme/_list-group.scss

```diff
@@ -107,6 +115,12 @@
 		}
 	}
 
+	.list-group-item-top-align {
```

Comment:
.list-group-item-top-align instead of creating this class, it looks like you can set the whole list-group to vertically align to the top.